src/webapp/components/RequestPopup.js

```diff
@@ -12,6 +12,7 @@ import HouseholdSizeChip from "./HouseholdSizeChip";
 import DrivingClusterChip from "./DrivingClusterChip";
 import { daysSinceSlackMessage } from "../helpers/time";
 import ClusterMapContext from "../context/ClusterMapContext";
+import { fields } from "../../lib/airtable/tables/requests";
```

Comment:
Same here: the absolute path (with or without tilde) didn't work.

Reply:
Possibly this is old news, but after looking over it, it looks like the `~whatever` module aliases aren't in use anywhere in the `webapp` directory... surprising! But I think that at least means that this problem isn't because of any change in this PR.

Reply:
Also, it looks like most of the other scripts in our `package.json` are requiring `module-alias/register` which I assume is what's giving other scripts the ability to resolve those aliases. However, I'm not actually sure how to do that in `local:react` since it runs `react-scripts` instead of `node`.